### DISCUSSION TITLE
Is it possible to create a new workspace via structurizr/cli?

### DISCUSSION THREAD

**Discussion by lekhanh1234** [structurizr/cli#133] [Q&A] [answered]
### Question

Hey there! I have some questions related to creating workspaces using structurizr/cli.

1. Is it possible to create a new workspace using structurizr/cli?
2. If not, what are the alternative ways to create a new workspace?
3. Could you also share some insights on whether automatically creating workspaces is considered an anti-pattern?

**Comment by simonbrowndotje** [comment, marked as answer]
1. No.
2. (and 3.) See https://github.com/structurizr/cli/discussions/128#discussioncomment-6315419
